lib/fluent/plugin/filter_filter_list.rb

```diff
@@ -1,21 +1,24 @@
 require 'fluent/plugin/out_filter_list/version'
-require 'aho_corasick'
+require 'matcher'
 require 'fluent/plugin/filter'
+require 'ip'
 
 module Fluent
   module Plugin
     class FilterListFilter < Filter
       include Matchers
+      include IP
 
       Plugin.register_filter('filter_list', self)
 
+      config_param :filter, :string, default: 'AC'
```

Comment:
Please describe the new configuration on README.

Reply:
The format of the list of IP addresses also should be described. Especially, the list accepts both usual IP address and a CIDR-notated group of IP addresses as an entry.